frontend/lib/store/tests/README.md

```diff
@@ -44,7 +44,7 @@ To run the tests in this project, follow these steps:
 1. **Install Dependencies**  
    Ensure all dependencies are installed. If not, run:
     ```bash
-    npm install
+    bun i
```

Comment:
if install use bun.   the `npm run test`  should also use `bun run test`.  I think we could give both, let the developers choose by themselves